usaspending_api/search/delta_models/subaward_search.py

```diff
@@ -461,22 +465,40 @@
         fpds.product_or_service_code,
         psc.description AS product_or_service_description,
 
+        LATEST_CURRENT_CD.recipient_location_congressional_code_current AS legal_entity_congressional_current,
         COALESCE(UPPER(bs.sub_legal_entity_country_code), 'USA') AS sub_legal_entity_country_code,
         rcc.country_name AS sub_legal_entity_country_name,
+        -- For county codes, we need to make sure we extract the right digits out and ensure it's zero-padded to 3.
         LPAD(CAST(CAST(REGEXP_EXTRACT(rec.county_numeric, '^[A-Z]*(\\d+)(?:\\.\\d+)?$', 1) AS SHORT) AS STRING), 3, '0') AS sub_legal_entity_county_code,
         rec.county_name AS sub_legal_entity_county_name,
         LEFT(COALESCE(bs.sub_legal_entity_zip, ''), 5) AS sub_legal_entity_zip5,
         rec.census_code AS sub_legal_entity_city_code,
+        -- For congressional districts, we need to make sure we extract the right digits out and ensure it's zero-padded to 2.
         LPAD(CAST(CAST(REGEXP_EXTRACT(UPPER(bs.sub_legal_entity_congressional), '^[A-Z]*(\\d+)(?:\\.\\d+)?$', 1) AS SHORT) AS STRING), 2, '0') AS sub_legal_entity_congressional,
+        (CASE
+            WHEN (
+                UPPER(bs.sub_legal_entity_country_code) <> 'USA'
+            ) THEN NULL
+            ELSE COALESCE(rl_cd_state_grouped.congressional_district_no, rl_zips.congressional_district_no, rl_cd_zips_grouped.congressional_district_no, rl_cd_city_grouped.congressional_district_no, rl_cd_county_grouped.congressional_district_no)
+        END) AS sub_legal_entity_congressional_current,
 
+        LATEST_CURRENT_CD.pop_congressional_code_current AS place_of_performance_congressional_current,
         fabs.place_of_performance_scope AS place_of_perform_scope,
         COALESCE(UPPER(bs.sub_place_of_perform_country_co), 'USA') AS sub_place_of_perform_country_co,
         pcc.country_name AS sub_place_of_perform_country_name,
+        -- For county codes, we need to make sure we extract the right digits out and ensure it's zero-padded to 3.
```

Comment:
I'm hesitant about repeating the `3` here. If the code changes we are also going to have to remember to change this comment too. That's two places an update needs to happen.

>  we need to make sure we extract the right digits out and ensure it's zero-padded to 3.

It's clear from looking at the code that this is happening so this may be knowledge duplication. Perhaps it'd be better if this comment answered why? If someone wants to know what, all they have to do is look at the code.

Reply:
> "Why are we using this regex and left padding for county codes?" If someone wants to know what, all they have to do is look at the code.

This ticket has nothing to do with that question. It's how we've always extracted congressional district code from the field (get the two digits, zero pad it). I don't know what more to explain. Do we just want to remove these regex comments?